vss-accessor/build.rs

```diff
@@ -0,0 +1,21 @@
+extern crate prost_build;
+
+use std::fs;
+use std::fs::File;
+use std::path::Path;
+
+fn main() {
+    download_file("https://raw.githubusercontent.com/lightningdevkit/vss-server/main/app/src/main/proto/vss.proto",
+                  "src/proto/vss.proto").unwrap();
```

Comment:
Eh, I don't think we want a dependency on github. Doesn't feel very decentralized. 😛 Would it have been better to make a single git repo containing both client and server that shared the proto file (via a symlink if needed)?

Reply:
I understand but this isn't really a real dependency on github. 
Its kind of equivalent to having code hosting dependency on github. 
We can change this URL anytime to something else.
Server side is java repo managed by gradle, cannot depend on same lib.

Another way would be bundle client with its own vss.proto but that too has its side-effects of maintaining same version across server-client.

Reply:
> Another way would be bundle client with its own vss.proto but that too has its own side-effects of maintaining same version across server-client.

Why do the client and server need to be in separate repositories?